docs/ref/settings.txt

```diff
@@ -641,6 +641,20 @@ PostgreSQL), it is an error to set this option.
 
 When :setting:`USE_TZ` is ``False``, it is an error to set this option.
 
+.. setting:: DATABASE-ENABLE_SERVER_SIDE_CURSORS
+
+``ENABLE_SERVER_SIDE_CURSORS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Default: ``True``
+
+Set this to ``False`` if you want to globally disable the use of server-side
+cursors with :meth:`~django.db.models.query.QuerySet.iterator`.
+
+This is a PostgreSQL-specific setting.
```

Comment:
This doesn't have an effect on Oracle? The `QuerySet.iterator()` docs don't make that clear.